Pipfile

```diff
@@ -0,0 +1,17 @@
+[[source]]
```

Comment:
There is already a `requirements.txt` in the repo. Perhaps we can remove that if the Pipfile is added, and update the running instructions in the README.

Reply:
Go ahead and remove the Pipfile.